docs/proposals/PROPOSAL_F.md

```diff
@@ -0,0 +1,374 @@
+# Proposal F
+
+OCI Index references it all
+
+## Description
+
+This proposal cherry picks ideas from [Proposal C](https://github.com/opencontainers/wg-reference-types/blob/main/docs/proposals/PROPOSAL_C.md) and [Proposal D](https://github.com/opencontainers/wg-reference-types/blob/main/docs/proposals/PROPOSAL_D.md).
+A **nested** [OCI Index](https://github.com/opencontainers/image-spec/blob/main/image-index.md) (an OCI Index referencing another OCI index, allowed by the Image Specification) is used as a top-level object to reference all the artifacts attached to an image.
+No change to the reference is expected, but a **well-known annotations** section is added.
+
+## Links
+
+| Description    | Link                                                 |
+| -------------- | ---------------------------------------------------- |
+| OCI Artifacts  | [View](https://github.com/opencontainers/artifacts)  |
+| Image Spec     | [View](https://github.com/opencontainers/image-spec) |
+
+## Modifications
+
+### JSON Schema
+
+The image-spec json schema is left unchanged.
+
+The OCI Index object references OCI image manifests and OCI artifacts attached to them, using OCI descriptors.
+OCI Artifacts descriptors **SHOULD** contain **reference** annotations, to expose the reference type and the image referenced.
+OCI Manifest descriptors **SHOULD** contain **well-known** annotations, to expose metadata attached to the image.
+
+#### Pre-defined annotations
+
+Reference annotations are added to the [Annotations](https://github.com/opencontainers/image-spec/blob/main/annotations.md) section.
+* `org.opencontainers.reference.digest`: digest referencing an image. The digest **SHOULD** be referenced as OCI Descriptor in the root OCI index **OR** in the nested OCI index to avoid a weak reference.
```

Comment:
Should this be worded as "SHOULD reference an OCI Descriptor"?

Reply:
What happens when an artifact/image's reference annotation points to a nested index which references the same artifact/image? Is there any way a client can resolve this?

Reply:
> What happens when an artifact/image's reference annotation points to a nested index which references the same artifact/image? Is there any way a client can resolve this?

What would be the use case ? I don't really get the workflow that would produce it 🤔

Reply:
> Should this be worded as "SHOULD reference an OCI Descriptor"?

My intent was to enforce the image producers to make sure that if they reference a digest with this annotation, they also reference it using an OCI Descriptor, otherwise it's only a weak pointer and nothing prevents the registry from garbage collecting the referenced digest.